src/code/z_view.c

```diff
@@ -51,80 +51,88 @@ void View_Init(View* view, GraphicsContext* gfxCtx) {
     view->up.z = 0.0f;
     view->eye.z = -1.0f;
 
-    if (D_8012ABF0) {
-        if (D_8012ABF0 == 0) {}
+    if (sViewNotInitialized) {
+        if (sViewNotInitialized == 0) {}
         osSyncPrintf("\nview: initialize ---\n");
-        D_8012ABF0 = false;
+        sViewNotInitialized = false;
     }
```

Comment:
You're right, I somehow totally blanked on the fact that this isn't per-view. I've renamed to `sLogOnNextViewInit`, since that's literally all it does.